# -----------------------------------------------------------------------------------
# variable names used in dmp_trajectory.cpp
# -----------------------------------------------------------------------------------

trajectory:

  controller_right_arm:  r_arm_x r_arm_xd r_arm_xdd r_arm_y r_arm_yd r_arm_ydd r_arm_z r_arm_zd r_arm_zdd r_arm_roll r_arm_rolld r_arm_rolldd r_arm_pitch r_arm_pitchd r_arm_pitchdd r_arm_yaw r_arm_yawd r_arm_yawdd r_shoulder_pan r_shoulder_pan_d r_shoulder_pan_dd r_shoulder_lift r_shoulder_lift_d r_shoulder_lift_dd r_upper_arm_roll r_upper_arm_roll_d r_upper_arm_roll_dd r_elbow_flex r_elbow_flex_d r_elbow_flex_dd r_forearm_roll r_forearm_roll_d r_forearm_roll_dd r_wrist_flex r_wrist_flex_d r_wrist_flex_dd r_wrist_roll r_wrist_roll_d r_wrist_roll_dd
  cart_and_joint_right_arm:  r_arm_x r_arm_xd r_arm_xdd r_arm_y r_arm_yd r_arm_ydd r_arm_z r_arm_zd r_arm_zdd r_arm_q0 r_arm_q0d r_arm_q0dd r_arm_q1 r_arm_q1d r_arm_q1dd r_arm_q2 r_arm_q2d r_arm_q2dd r_arm_q3 r_arm_q3d r_arm_q3dd  r_shoulder_pan r_shoulder_pan_d r_shoulder_pan_dd r_shoulder_lift r_shoulder_lift_d r_shoulder_lift_dd r_upper_arm_roll r_upper_arm_roll_d r_upper_arm_roll_dd r_elbow_flex r_elbow_flex_d r_elbow_flex_dd r_forearm_roll r_forearm_roll_d r_forearm_roll_dd r_wrist_flex r_wrist_flex_d r_wrist_flex_dd r_wrist_roll r_wrist_roll_d r_wrist_roll_dd

  debug_cart_right_arm: time dt unspecified r_arm_x r_arm_xd r_arm_xdd r_arm_y r_arm_yd r_arm_ydd r_arm_z r_arm_zd r_arm_zdd r_arm_q0 r_arm_q0d r_arm_q0dd r_arm_q1 r_arm_q1d r_arm_q1dd r_arm_q2 r_arm_q2d r_arm_q2dd r_arm_q3 r_arm_q3d r_arm_q3dd
  cart_right_arm: r_arm_x r_arm_xd r_arm_xdd r_arm_y r_arm_yd r_arm_ydd r_arm_z r_arm_zd r_arm_zdd r_arm_q0 r_arm_q0d r_arm_q0dd r_arm_q1 r_arm_q1d r_arm_q1dd r_arm_q2 r_arm_q2d r_arm_q2dd r_arm_q3 r_arm_q3d r_arm_q3dd
  r_arm_x:
    name: r_arm_x
    unit: m
  r_arm_xd:
    name: r_arm_xd
    unit: m/s
  r_arm_xdd:
    name: r_arm_xdd
    unit: m/s^2
  r_arm_y:
    name: r_arm_y
    unit: m
  r_arm_yd:
    name: r_arm_yd
    unit: m/s
  r_arm_ydd:
    name: r_arm_ydd
    unit: m/s^2
  r_arm_z:
    name: r_arm_z
    unit: m
  r_arm_zd:
    name: r_arm_zd
    unit: m/s
  r_arm_zdd:
    name: r_arm_zdd
    unit: m/s^2
  r_arm_q0:
    name: r_arm_q0
    unit: "-"
  r_arm_q0d:
    name: r_arm_q0d
    unit: "-"
  r_arm_q0dd:
    name: r_arm_q0dd
    unit: "-"
  r_arm_q1:
    name: r_arm_q1
    unit: "-"
  r_arm_q1d:
    name: r_arm_q1d
    unit: "-"
  r_arm_q1dd:
    name: r_arm_q1dd
    unit: "-"
  r_arm_q2:
    name: r_arm_q2
    unit: "-"
  r_arm_q2d:
    name: r_arm_q2d
    unit: "-"
  r_arm_q2dd:
    name: r_arm_q2dd
    unit: "-"
  r_arm_q3:
    name: r_arm_q3
    unit: "-"
  r_arm_q3d:
    name: r_arm_q3d
    unit: "-"
  r_arm_q3dd:
    name: r_arm_q3dd
    unit: "-"

  r_arm_roll:
    name: r_arm_roll
    unit: rad
  r_arm_rolld:
    name: r_arm_rolld
    unit: rad/s
  r_arm_rolldd:
    name: r_arm_rolldd
    unit: rad/s^2  
  r_arm_pitch:
    name: r_arm_pitch
    unit: rad
  r_arm_pitchd:
    name: r_arm_pitchd
    unit: rad/s
  r_arm_pitchdd:
    name: r_arm_pitchdd
    unit: rad/s^2  
  r_arm_yaw:
    name: r_arm_yaw
    unit: rad
  r_arm_yawd:
    name: r_arm_yawd
    unit: rad/s
  r_arm_yawdd:
    name: r_arm_yawdd
    unit: rad/s^2
    
  debug_pr2_r_arm_joint_names: time dt unspecified r_shoulder_pan r_shoulder_pan_d r_shoulder_pan_dd r_shoulder_lift r_shoulder_lift_d r_shoulder_lift_dd r_upper_arm_roll r_upper_arm_roll_d r_upper_arm_roll_dd r_elbow_flex r_elbow_flex_d r_elbow_flex_dd r_forearm_roll r_forearm_roll_d r_forearm_roll_dd r_wrist_flex r_wrist_flex_d r_wrist_flex_dd r_wrist_roll r_wrist_roll_d r_wrist_roll_dd
  pr2_r_arm_joint_names: r_shoulder_pan r_shoulder_pan_d r_shoulder_pan_dd r_shoulder_lift r_shoulder_lift_d r_shoulder_lift_dd r_upper_arm_roll r_upper_arm_roll_d r_upper_arm_roll_dd r_elbow_flex r_elbow_flex_d r_elbow_flex_dd r_forearm_roll r_forearm_roll_d r_forearm_roll_dd r_wrist_flex r_wrist_flex_d r_wrist_flex_dd r_wrist_roll r_wrist_roll_d r_wrist_roll_dd
  r_shoulder_pan:
    name: r_shoulder_pan
    unit: rad
  r_shoulder_pan_d:
    name: r_shoulder_pan_d
    unit: rad/s
  r_shoulder_pan_dd:
    name: r_shoulder_pan_dd
    unit: rad/s^2
  r_shoulder_lift:
    name: r_shoulder_lift
    unit: rad
  r_shoulder_lift_d:
    name: r_shoulder_lift_d
    unit: rad/s
  r_shoulder_lift_dd:
    name: r_shoulder_lift_dd
    unit: rad/s^2
  r_upper_arm_roll:
    name: r_upper_arm_roll
    unit: rad
  r_upper_arm_roll_d:
    name: r_upper_arm_roll_d
    unit: rad/s
  r_upper_arm_roll_dd:
    name: r_upper_arm_roll_dd
    unit: rad/s^2
  r_elbow_flex:
    name: r_elbow_flex
    unit: rad
  r_elbow_flex_d:
    name: r_elbow_flex_d
    unit: rad/s
  r_elbow_flex_dd:
    name: r_elbow_flex_dd
    unit: rad/s^2
  r_forearm_roll:
    name: r_forearm_roll
    unit: rad
  r_forearm_roll_d:
    name: r_forearm_roll_d
    unit: rad/s
  r_forearm_roll_dd:
    name: r_forearm_roll_dd
    unit: rad/s^2
  r_wrist_flex:
    name: r_wrist_flex
    unit: rad
  r_wrist_flex_d:
    name: r_wrist_flex_d
    unit: rad/s
  r_wrist_flex_dd:
    name: r_wrist_flex_dd
    unit: rad/s^2
  r_wrist_roll:
    name: r_wrist_roll
    unit: rad
  r_wrist_roll_d:
    name: r_wrist_roll_d
    unit: rad/s
  r_wrist_roll_dd:
    name: r_wrist_roll_dd
    unit: rad/s^2
